script.js

```diff
@@ -1,12 +1,115 @@
 //You can edit ALL of the code here
+// function setup() {
+//   const allEpisodes = getAllEpisodes();
+//   makePageForEpisodes(allEpisodes);
+// }
+
+// function makePageForEpisodes(episodeList) {
+//   const rootElem = document.getElementById("root");
+//   rootElem.textContent = `Got ${episodeList.length} episode(s)`;
+// }
+
+// window.onload = setup;
+////////////////
+
+//////////////////
+const searchBox = document.getElementById("search-box");
+const searchCount = document.getElementById("search-count");
+
+//You can edit ALL of the code here
+function getLiveEpisodes() {
+  var newVar = fetch("https://api.tvmaze.com/shows/82/episodes")
+    .then((response) => response.json())
+    .then((data) => {
+      console.log("This is data", data);
+      // data.forEach((e) => {
+      //   const episode = document.createElement("div");
+      //   const heading = document.createElement("h3");
+      //   const episodeImage = document.createElement("img");
+      //   const summary = document.createElement("p");
+
+      //   heading.innerText = `${e.name} - ${formatSeriesAndEpisode(
+      //     e.season,
+      //     e.number
+      //   )}`;
+
+      //   summary.innerHTML = e.summary;
+      //   episodeImage.src = e.image.medium;
+      //   episode.className = "episode";
+
+      //   episode.appendChild(heading);
+      //   episode.appendChild(episodeImage);
+      //   episode.appendChild(summary);
+      //   episodeContainer.appendChild(episode);
+      // });
+    });
+}
+
 function setup() {
-  const allEpisodes = getAllEpisodes();
-  makePageForEpisodes(allEpisodes);
+  // const allEpisodes = getLiveEpisodes();
+
+  searchBox.addEventListener("keyup", onSearchKeyUp);
+  fetch("https://api.tvmaze.com/shows/82/episodes")
+    .then((response) => response.json())
+    .then((data) => {
+      console.log("This is data", data);
+      makePageForEpisodes(data);
+    });
 }
 
 function makePageForEpisodes(episodeList) {
-  const rootElem = document.getElementById("root");
```

Comment:
You need to save the episode list so you can use it in `onSearchKeyUp` function as this is not currently working